crypto/src/test/java/civitas/crypto/proofdvr/ConstructElGamalProofDVRTest.java

```diff
@@ -10,27 +10,29 @@
 
 import civitas.common.TestBase;
 
-public class ConstructElGamalProofDVRTest extends TestBase
+class ConstructElGamalProofDVRTest extends TestBase
 		implements ElGamalProofDVRTestData {
 
 	@InjectMocks
 	ConstructElGamalProofDVR constructElGamalProofDVR;
 
 	//@formatter:off
```

Comment:
The formatter markings were there to keep the display name nice. They are not needed now.